src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/CodeFlowConflictResolver.cs

```diff
@@ -83,52 +131,130 @@ await repo.CommitAsync(
         }
     }
 
+    /// <summary>
```

Comment:
should this description be above the `TryResolvingConflictWithCrossingFlow` method? couse this one is also handling version files